### PR TITLE
HMIS-1240 VH Cancel endpoint

### PR DESCRIPTION
**Pull request by junaidiqbalmoj**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/HMIS-1240

### Change description ###

Add missing endpoint in HMI APIM about video hearing cancellation. 

**Does this PR introduce a breaking change?** (check one with "x")
[ ] Yes
[x] No

### CODE REVIEW

File: infrastructure/template/api-op-vh-cancel-hearing-policy.xml [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+                            <JObject>()["value"].ToString())
+                        </value>
+                    </set-header>
+                    <set-body>@{
+                        return new JObject(
+                        new JProperty("assignment_group", "56b756774fbd368057db0b318110c7bd"),
+                        new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
+                        new JProperty("category", "Data Issue"),
+                        new JProperty("contact_type", "Alert"),
+                        new JProperty("description", "HMI - Update VH By ID: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
```
Comment (joshblackmoor) [inline infrastructure/template/api-op-vh-cancel-hearing-policy.xml:111]: Maybe change description to make it clear its the cancel endpoint

File: infrastructure/template/api-op-vh-cancel-hearing-policy.xml [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
```
Comment (joshblackmoor) [inline infrastructure/template/api-op-vh-cancel-hearing-policy.xml:156]: same here

File: infrastructure/template/api-op-vh-cancel-hearing-policy.xml [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+                            <JObject>()["value"].ToString())
+                        </value>
+                    </set-header>
+                    <set-body>@{
+                return new JObject(
+                new JProperty("assignment_group", "56b756774fbd368057db0b318110c7bd"),
+                new JProperty("caller_id", "1475a0491b743414f0dc85e4464bcb7a"),
+                new JProperty("category", "Data Issue"),
+                new JProperty("contact_type", "Alert"),
+                new JProperty("description", "HMI - Update VH By ID: " + context.Request.Headers.GetValueOrDefault("transactionIdHMCTS","")),
```
Comment (joshblackmoor) [inline infrastructure/template/api-op-vh-cancel-hearing-policy.xml:200]: same here

File: infrastructure/template/api-op-vh-cancel-hearing-policy.xml (hunk 1 above)
Comment (junaidiqbalmoj) [reply infrastructure/template/api-op-vh-cancel-hearing-policy.xml]: fixed.

File: infrastructure/template/api-op-vh-cancel-hearing-policy.xml (hunk 3 above)
Comment (junaidiqbalmoj) [reply infrastructure/template/api-op-vh-cancel-hearing-policy.xml]: fixed.

File: infrastructure/template/api-op-vh-cancel-hearing-policy.xml (hunk 2 above)
Comment (junaidiqbalmoj) [reply infrastructure/template/api-op-vh-cancel-hearing-policy.xml]: fixed.

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**